yt/testing.py

```diff
@@ -478,6 +478,51 @@ def fake_sph_orientation_ds():
 
     return load_particles(data=data, length_unit=1.0, bbox=bbox)
 
+def fake_sph_grid_ds():
```

Comment:
seems useful :)

Reply:
yeah what did you think about an arg with a default for setting smoothing lengths or the size of the grid? didn't want to over complicate but might be useful for further tests that way

Reply:
I’d say add it when you need it. We shouldn’t add code that doesn’t have users inside yt yet.